rocky/rocky/views/organization_edit.py

```diff
@@ -1,24 +1,28 @@
 from django.contrib import messages
 from django.contrib.auth.mixins import PermissionRequiredMixin
-from django.shortcuts import redirect
 from django.urls.base import reverse
 from django.utils.translation import gettext_lazy as _
 from django.views.generic import UpdateView
 from django_otp.decorators import otp_required
 from two_factor.views.utils import class_view_decorator
 
-from account.forms import OrganizationForm
+from account.forms import OrganizationUpdateForm
 from tools.models import Organization
 
 
 @class_view_decorator(otp_required)
 class OrganizationEditView(PermissionRequiredMixin, UpdateView):
-    form_class = OrganizationForm
+    form_class = OrganizationUpdateForm
```

Comment:
@underdarknl here I used the regular `OrganizationUpdateForm` and for the onboading I used `OnboardingOrganizationUpdateForm `